### PR TITLE
Pull upstream.  Modernize a bit.

### LINKED ISSUE #4: GitHub Actions
# 🚀 Feature Proposal

Transfer to GitHub Actions

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description

- Pull in upstream CI changes.
- Modernize tests and container creation.
- Fixes #4 

## 💭 Motivation and Context

Keeping the bit rot at bay.

## 🧪 Testing

All tests executed.  New tests added for version checks.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
:rocket:

**Review by dav3r — APPROVED** [review]
🤽‍♂

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 # postfix-docker 📮🐳 #
 
 [![Build Status](https://travis-ci.com/cisagov/postfix-docker.svg?branch=develop)](https://travis-ci.com/cisagov/postfix-docker)
```
Comment (mcdonnnj) [inline README.md]: Do we still need this?

**Review by mcdonnnj — APPROVED** [review]
Looking good, but please see my comment.

File: README.md (hunk 1 above)
Comment (felddy) [reply README.md]: Yup.  That one snuck through.  Good catch.  Removed in https://github.com/cisagov/postfix-docker/pull/5/commits/a4867e45c0d3daf1319b7f406eb1910e5c3dcf78.

**Pull request merged**